packages/web-api-scan-runner/src/runner/url-deduplicator.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { injectable } from 'inversify';
+import { uniq } from 'lodash';
+
+@injectable()
+export class UrlDeduplicator {
```

Comment:
I don't think this needs to be a class; it maintains no state.

Reply:
I was thinking that as well but I couldn't get it to inject using inversify (not supported I believe). After discussing with Kelsey, the way this is dealt with is by using default params. I think that could potentially lead to other issues in the future but for now, will change it to that.